changes/ocd-3790.sql

```diff
@@ -0,0 +1,69 @@
+DROP TABLE IF EXISTS openchpl.attestation_period_developer_exception;
+
+TRUNCATE openchpl.change_request_website,
+	openchpl.change_request_developer_details,
+	openchpl.change_request_attestation_submission,
+	openchpl.change_request_attestation_response,
+	openchpl.change_request_status,
+	openchpl.change_request,
+	openchpl.developer_attestation_response,
+	openchpl.developer_attestation_submission, 
+	openchpl.attestation_period RESTART IDENTITY;
+
+CREATE TABLE IF NOT EXISTS openchpl.attestation_period_developer_exception (
+	id bigserial NOT NULL,
+	attestation_period_id bigint NOT NULL,
+	developer_id bigint NOT NULL,
```

Comment:
Since this has a reference to developer_id, I was expecting to see something in the soft_delete triggers file.